src/agor/tools/agent_coordination.py

```
@@ -69,46 +96,98 @@ def discover_current_situation(self, agent_id: Optional[str] = None) -> Dict:
+        # Strategy type detection mapping
+        strategy_patterns = {
+            "Parallel Divergent Strategy": ("parallel_divergent", self._parse_parallel_divergent_strategy),
+            "Pipeline Strategy": ("pipeline", self._parse_pipeline_strategy),
+            "Swarm Strategy": ("swarm", self._parse_swarm_strategy),
+            "Red Team Strategy": ("red_team", lambda c: {"type": "red_team", "phase": "active"}),
+            "Mob Programming Strategy": ("mob_programming", lambda c: {"type": "mob_programming", "phase": "active"}),
+        }
+
+        for pattern, (_, parser) in strategy_patterns.items():
+            if pattern in content:
+                return parser(content)
```

Pylint (3.3.7)

[error] parser is not callable

(E1102)